src/components/MessageBox.js

```diff
@@ -11,55 +12,51 @@ const StyleUtils = require('../utils/Style');
 
 class MessageBox extends React.Component {
   static propTypes = {
-    children: PropTypes.node,
+    button: PropTypes.node,
     color: PropTypes.string,
     icon: PropTypes.string,
+    isSmall: PropTypes.bool,
+    message: PropTypes.node,
     styles: PropTypes.object,
     theme: themeShape,
     title: PropTypes.string
   };
 
-  state = {
-    isOpen: true
-  };
-
-  _toggleMessageBox = () => {
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
-  };
-
   render () {
     const theme = StyleUtils.mergeTheme(this.props.theme);
     const styles = this.styles(theme);
 
     return (
       <div className='mx-message-box' style={styles.component}>
-        <div onClick={this.props.children ? this._toggleMessageBox : () => {}} style={styles.header}>
-          <div style={styles.leftHeader}>
-            <Icon
-              size={20}
-              style={Object.assign({}, styles.icon, { marginRight: theme.Spacing.SMALL })}
-              type={this.props.icon}
-            />
-            <div style={styles.title}>{this.props.title}</div>
+        <div style={styles.alertbar} />
+        <div style={styles.messageSection}>
+          <div style={styles.messageBody}>
+            <div>
+              <Icon
+                size={20}
+                style={Object.assign({}, styles.icon, { marginRight: theme.Spacing.SMALL })}
```

Comment:
I know this is already existing code but since you are here can we kill `Object.assign` in favor of object spread syntax?

Reply:
yep, I can absolutely do that